app/utils/discogs.schemas.ts

```diff
@@ -1,8 +1,10 @@
 import z from "zod";
 
 const AlbumSchema =  z.object({
+  cover_image:z.string(),
```

Comment:
properties on objects should almost always be camelCase

```suggestion
  coverImage: z.string(),
```

Reply:
if discogs isn't returning it to you this way, I'd adjust it in a .map (you can massage the data to match how **you** want it to loo)

Reply:
Modified the key but getting a type error for album in my map method in discogs. Tried to assign type AlbumSchema to album but cover_image doesn't exist on it now so that throws an error. This is also an example of a time where I don't understand why I have to explicitly define the type of album where as in other cases I don't.